source/integrate-with-integration-environment/manage-your-service-s-configuration.html.md.erb

```diff
@@ -46,6 +46,8 @@ To register your service, you must send:
 * the URL you’d like your users redirected to if they log out of your service - if you do not specify one, your users will be redirected to the [default GOV.UK sign out page](https://signin.account.gov.uk/signed-out)
 * your `sector_identifier_uri` with the identifier for your sector
 
+By default, GOV.UK One Login will sign the `id_token` JSON Web Token (JWT) using the `ES256` algorithm but some third party tooling does not support `ES256`. If your service needs an alternative algorithm, we can sign your JWT `id_token` using the `RSA256` algorithm. Let us know if you need this when you register your service.
```

Comment:
Minor thing, but you reverse this phrase within this paragraph:

> `id_token` JSON Web Token (JWT)

and

> JWT `id_token`

I'd stick with "`id_token` JWT".

Reply:
I don't know if this comment is clear. Let me know if it isn't!

Reply:
good catch! Updating now